frontend/user-bookmarks/style.css

```
@@ -0,0 +1,313 @@
+    --emerald: #10b981;
+    --ivory: #faf8f1;
+    --terracotta: #d97706;
+    --muted: #6b7280;
+    --dark: #111827;
+    --card-bg: rgba(250, 248, 241, 0.8);
+    --glass-bg: rgba(250, 248, 241, 0.6);
+}
+
+/* Page Background */
+body {
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)